app/javascript/src/components/Dashboard/Notes/NotesCard/CardsDropdown.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const { Menu, MenuItem } = Dropdown;
+
+const CardsDropdown = ({ note, setShowDeleteAlert, setSelectedNoteId }) => {
+  const OnDelete = () => {
```

Comment:
You are not paying attention to the naming. This should have been caught in your self-review process. Go through https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process